cypress/support/getSubjects.js

```diff
@@ -0,0 +1,22 @@
+const subjects = [
+  'English',
+  'Computer Science',
+  'Commerce',
+  'Economics',
+  'Accounting',
+  'Maths',
+  'Arts',
+  'Social Studies',
+  'Physics',
+  'Chemistry',
+  'Civics',
+  'History',
+  'Biology',
+  'Hindi'
+];
+
+const getSubjects = () => {
+  return subjects.filter(() => Math.random() * 10 > 5);
```

Comment:
The filter function currently uses a random condition to select subjects, which may not be the intended logic. If the goal is to randomly select a subset of subjects, consider using a more controlled approach to ensure the desired number of subjects is returned.